src/tests/Favorites.test.js

```diff
@@ -8,9 +8,11 @@ describe('Favorites', () => {
       render(<Favorites />);
     });
 
-    // test('must show favorite videos', () => {
-    //   render(<Favorites />);
-    // });
+    test('must be able to retrieve from local storage', () => {
+      render(<Favorites />);
+      const items = { ...localStorage };
+      expect(items).toBeTruthy();
+    });
 
 
```

Comment:
Probably you can add 2 test, one to validate that if you pass at least a video it renders something (you can check title, description or whatever is esasy to validate), and second test will be to validate that when your array is empty you dont render anything